src/main/resources/reference.conf

```diff
@@ -37,6 +39,9 @@ akka {
   loggers = ["akka.event.slf4j.Slf4jLogger"]
 }
 
+akka.http.server.idle-timeout = 95 s # Akka default is 60 s
+akka.http.server.request-timeout = 65 s # Akka default is 20 s
```

Comment:
I recall something about these settings not working unless they are in agora.conf

Reply:
I'll test this. I verified by outputting the value from inside Agora code ... but that doesn't mean they're actually working. I'll leave them here and add to firecloud-develop if necessary.